tests/QARegressionTests/test_Issues/test_CoreIssue208.py

```diff
@@ -0,0 +1,83 @@
+import json
+import os
+import subprocess
+import unittest
+import pytest
+
+"""This regression test is for automating the validation of acceptancce criteria

```

Comment:
Can you include what the expected acceptance criteria is in these tests

Reply:
updated